minimal-notebook/notebook.conf

```diff
@@ -1,18 +1,10 @@
 [program:notebook]
 user=jovyan
```

Comment:
Would be nice except supervisor doesn't respect env vars in the user field as @costerwi pointed out in PR #20.